di-ipv-cimit-stub/lambdas/put-contra-indicators/src/main/java/uk/gov/di/ipv/core/putcontraindicators/service/ContraIndicatorsService.java

```diff
@@ -161,6 +161,7 @@ private void saveOrUpdateCimitStubItems(String userId, List<CimitStubItem> cimit
                         cimitStubItemService.persistCimitStub(
                                 userId,
                                 cimitStubItem.getContraIndicatorCode().toUpperCase(),
+                                cimitStubItem.getIssuers(),
                                 cimitStubItem.getIssuanceDate(),
                                 Collections.emptyList());
                     } else {
```

Comment:
Do we need to update the `else` branch here to combine the issuers from `cimitStubItem` and `dbCimitStubItem` (like in the management API)?